ccd-definition/AuthorisationCaseField.json

```diff
@@ -482,6 +496,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "respondents1",
+    "UserRole": "caseworker-publiclaw-courtadmin",
```

Comment:
I guess you did not give same to gatekeeper role because no-one is using it in production yet and when we rename 'respondents1' to 'respondents' it will work again as it was.

Reply:
less unpicking - you happy with this?

Reply:
Happy. I like your pragmatic approach @swalker125 ;)